config/locales/hyrax.en.yml

```diff
@@ -575,7 +575,7 @@ en:
             collection_title: "Collection Title"
           this_collection_in_other_collections_description: "You can add this collection to other collections as a sub-collection."
         new:
-          header: "New %{type_title}"
+          header: "New"
```

Comment:
Don't think this change will be needed if you take out the changes to app/views/hyrax/dashboard/collections/new.html.erb.

Reply:
Okay, reverted. It is a bug but I can fix it for 2071 if you want or we could make it a separate issue.